source/tyr/tyr/binarisation.py

```diff
@@ -715,31 +714,28 @@ def openaddresses2mimir(self, autocomplete_instance, filename, job_id, dataset_u
 @celery.task(bind=True)
 def osm2mimir(self, autocomplete_instance, filename, job_id, dataset_uid):
     """ launch osm2mimir """
+
     autocomplete_instance = models.db.session.merge(autocomplete_instance)  # reatache the object
     logger = get_autocomplete_instance_logger(autocomplete_instance, task_id=job_id)
     logger.debug('running osm2mimir for {}'.format(job_id))
     job = models.Job.query.get(job_id)
     cnx_string = current_app.config['MIMIR_URL']
     working_directory = unzip_if_needed(filename)
 
-    params = ['-i', working_directory, '--connection-string', cnx_string]
-    for lvl in autocomplete_instance.admin_level:
-        params.append('--level')
```

Comment:
It means old parameters like '--level' will not be used anymore?

Reply:
yes :+1: